docker/server-jetty/src/main/server-jetty/requirements.txt

```diff
@@ -1,5 +1,6 @@
-adbc-driver-manager==0.1.0
-adbc-driver-postgresql==0.1.0
+adbc-driver-manager==0.2.0
+adbc-driver-postgresql==0.2.0
+connectorx==0.3.1
```

Comment:
I think adding this requirement will cause M1 integration tests to fail. I think we can hack in the same platform check that we've added for the base images...